Dockerfile

```diff
@@ -8,4 +8,6 @@ RUN go build -o ./openfga ./cmd/openfga
 FROM alpine as final
 EXPOSE 8080
 COPY --from=builder /app/openfga /app/openfga
-ENTRYPOINT ["/app/openfga"]
\ No newline at end of file
+COPY --from=builder /app/static /app/static
```

Comment:
Can we add `EXPOSE 3000` as well please?

Reply:
Done